java/amazon-kinesis-producer-sample/src/com/amazonaws/services/kinesis/producer/sample/SampleProducer.java

```diff
@@ -147,6 +151,7 @@ public void run() {
                 log.info(String.format(
                         "Put %d of %d so far (%.2f %%), %d have completed (%.2f %%)",
                         put, total, putPercent, done, donePercent));
+                log.info("Oldest future as of now is " + producer.getOldestFutureTimeFromNow());
```

Comment:
Should this be refactored to getOldestRecordTimeInSeconds ?

Reply:
Done